src/Client.php

```diff
@@ -21,7 +23,6 @@ class Client
     private $sessionFile;
     private $qrcodeCallback;
     private $logged = false;
-    private $running = false;
```

Comment:
This property is necessary to control the while loop

Reply:
I've reverted that change.  Could you explain why it cannot be changed to a local variable (as I did originally) as I don't understand.  Many thanks.